Description: creates blank kaji release
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+shinken-module-pickle-retention-file-generic (1.4-1kaji0.1) UNRELEASED; urgency=medium
+
+  * Kaji release, same package as 1.4-1
+
+ -- Matthieu Caneill <matthieu.caneill@savoirfairelinux.com>  Tue, 22 Jul 2014 14:29:34 -0400
+
 shinken-module-pickle-retention-file-generic (1.4-1) unstable; urgency=medium
 
   * New upstream version 1.4
